subsys/fb/cfb.c

```diff
@@ -343,7 +351,6 @@ int cfb_framebuffer_init(const struct device *dev)
 	fb->pixel_format = cfg.current_pixel_format;
 	fb->screen_info = cfg.screen_info;
 	fb->buf = NULL;
-	fb->font_idx = 0U;
```

Comment:
Is it need to remove?

Reply:
yes. same line twice. please see commit message.

Reply:
Sorry, I missed the comments.
It's no problem.